runtime/themes/heisenberg.toml

```diff
@@ -0,0 +1,80 @@
+# Author: IrishMaestro <github.com/irishmaestro>
+
+# Syntax highlighting
+"type" = { fg = "crystal_blue" }
+"type.builtin" = { fg = "crystal_blue" }
+"constructor" = { fg = "barium_green" }
+"constant" = { fg = "hazmat_yellow" }
+"string" = { fg = "crystal_blue" }
+"string.regexp" = { fg = "orange" }
+"string.special" = { fg = "vapor_yellow" }
+"comment" = { fg = "teddy_bear_pink", modifiers = ["slow_blink", "italic"] }
+"variable" = { fg = "element_green" }
+"variable.parameter" = { fg = "hazmat_yellow" }
+"label" = { fg = "vapor_yellow" }
+"punctuation" = { fg = "barium_green" }
+"keyword" = { fg = "barium_green" }
+"keyword.control" = { fg = "barium_green" }
+"keyword.directive" = { fg = "teddy_bear_pink" }
+"operator" = { fg = "hazmat_yellow" }
+"function" = { fg = "cash_green", modifiers = ["bold"] }
+"tag" = { fg = "crystal_blue" }
+"namespace" = { fg = "hazmat_yellow" }
+"markup.heading" = { fg = "hazmat_yellow" }
+"markup.list" = { fg = "vapor_yellow" }
+"markup.raw.block" = { bg = "gray", fg = "orange" }
+"markup.link.url" = { fg = "crystal_blue" }
+"markup.link.text" = { fg = "vapor_yellow" }
+"markup.link.label" = { fg = "barium_green" }
+"markup.quote" = { fg = "vapor_yellow" }
+"diff.plus" = { fg = "cash_green" }
+"diff.minus" = { fg = "chili_powder_red" }
+"diff.delta" = { fg = "cash_green" }
+
+# Interface
+"ui.background" = { bg = "background" }
+"ui.cursor" = { bg = "crystal_blue", fg = "background" }
+"ui.cursor.match" = { fg = "hazmat_yellow" }
+"ui.linenr" = { fg = "crystal_blue" }
+"ui.linenr.selected" = { fg = "barium_green" }
+"ui.statusline" = { fg = "crystal_blue", bg = "black" } 
+"ui.cursorline" = { bg = "background" }
+"ui.popup" = { fg = "crystal_blue", bg = "background" }
+"ui.window" = { fg = "barium_green" }
+"ui.help" = { fg = "crystal_blue", bg = "background"}
+"ui.text" = { fg = "crystal_blue" }
+"ui.text.focus" = { bg = "background", fg = "barium_green" }
+"ui.text.info" = { fg = "crystal_blue" }
+"ui.virtual.whitespace" = { fg = "dark_gray" }
+"ui.virtual.ruler" = { bg = "background" }
```

Comment:
Cursorline and rulers end up being invisible because they're set to the same color as the background. You need to enable config options these to see those two features: `:set cursorline true` and `:set rulers [80]`

Reply:
Good call. Updated `ui.cursorline` & `ui.virtual.ruler`